.github/workflows/tests.yaml

```diff
@@ -16,7 +16,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install prerequisites
-      run: ./install.sh
+      run: |
+        [[ $(uname) == Linux ]] && sudo apt-get install -y libpthread-stubs0-dev libc6-dev
```

Comment:
Why is this necessary? Could you point me to the failure?

Reply:
It's this:

```
/home/runner/miniforge3/envs/dgfem/bin/mpicc -pthread -B /home/runner/miniforge3/envs/dgfem/compiler_compat -Wl,--sysroot=/ _configtest.o -Lbuild/temp.linux-x86_64-3.8 -o _configtest
    /home/runner/miniforge3/envs/dgfem/compiler_compat/ld: cannot find /lib/libpthread.so.0
    /home/runner/miniforge3/envs/dgfem/compiler_compat/ld: cannot find /usr/lib/libpthread_nonshared.a
    collect2: error: ld returned 1 exit status
    failure.
    removing: _configtest.c _configtest.o
    error: Cannot link MPI programs. Check your configuration!!!
    ----------------------------------------
ERROR: Command errored out with exit status 1: /home/runner/miniforge3/envs/dgfem/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-iacl28uh/mpi4py/setup.py'"'"'; 
```